package dev.local.auth.exceptions;

public class UsernameTakenException {
}
